config/confighttp/confighttp.go

```diff
@@ -325,8 +336,13 @@ func WithDecoder(key string, dec func(body io.ReadCloser) (io.ReadCloser, error)
 	}
 }
 
-// ToServer creates an http.Server from settings object.
+// Deprecated: [v0.97.0] Use ToServerContext instead.
```

Comment:
Should be version where this deprecation is going to be released
```suggestion
// Deprecated: [v0.98.0] Use ToServerContext instead.
```